### DISCUSSION TITLE
Endre fargetema automatisk gjennom dagen

### DISCUSSION THREAD

**Discussion by forteller** [entur/tavla#549] [Ideas]
Hadde vært kult om man kunne velge at en tavle automatisk skulle ha lyst tema på dagen, Entur-temaet på kvelden og mørkt tema på natta. Kanskje enda bedre om det ikke er basert på rent klokkeslett, men på når sola går ned den dagen, slik at den skifter tidligere til Entur-temaet når det blir tidligere mørkt.

**Comment by lindtvedtsebastian** [comment]
This discussion is linked to the old (deprecated) version of Tavla and will therefore be closed.
